packages/client/src/views/RecordAudit/ActionDetailsModal.tsx

```diff
@@ -241,7 +241,7 @@ export const ActionDetailsModalListTable = ({
         columns={commentsColumn}
         content={actionDetailsData.comments
           .concat(actionDetailsData.statusReason?.text || [])
-          .map((text: string) => ({ comment: text }))}
+          .map((text: any) => ({ comment: text.comment }))}
```

Comment:
I'm guessing this bug was caused by incorrect typing. Wouldn't it make sense to fix the typing to ensure it doesn't happen again when someone changes the code?

Reply:
Yes. Absolutely make sense.